pyproject.toml

```diff
@@ -39,7 +39,7 @@ commands =
 
 [tool.poetry.dependencies]
 python = "^3.6.1"
-cognite-sdk-experimental = "^0.5"
+cognite-sdk-experimental = "^0."
```

Comment:
does this make a difference?

Reply:
yes, because 0.x is breaking in semantic versioning, it won't auto upgrade otherwise